react/components/BuyButton/index.js

```diff
@@ -1,15 +1,12 @@
 import find from 'lodash/find'
```

Comment:
⛏ ramda? :D

Reply:
What about do that when migrating to typescript?

Reply:
Forget it. If @luciannojunior is so in love with it, I'll do it 😍